src/components/feedback/modal-components/ModalHeader.tsx

```diff
@@ -55,7 +55,7 @@ const ModalHeader = ({
 		</ModalTitle>
 		{showCloseIcon && onClose && (
 			<Tooltip label={closeIconTooltip} disabled={!closeIconTooltip}>
-				<IconButton icon="Close" size="large" onClick={onClose} />
+				<Button icon="Close" size="large" onClick={onClose} />
```

Comment:
we need to add type={'ghost'} and color={'text'} while converting from IconButton to Button to not lose the defaults